pages/linux/chroot.md

```diff
@@ -0,0 +1,23 @@
+# chroot
+
+> Run command or interactive shell with special root directory.
+
+- General Syntax
+
+`chroot [OPTION] NEWROOT {{COMMAND [ARG...]}}`
+
+- specify user and group (ID or name) to use
+
+`chroot −−userspec={{USER:GROUP}}`
+
+- specify supplementary groups as g1,g2,..,gN
+
+`chroot −−groups={{G_LIST}}`
+
+- display help and exit
```

Comment:
Isn't this one a bit of a no-brainer? Usually people replace `command --help` with `tldr command`, no need to send them back. Same goes for the `--version` below.